### DISCUSSION TITLE
Meeting August 28, 2023

### DISCUSSION THREAD

**Discussion by RCODI** [opendigital/opendigital-org#44] [General]
## Review of action items from last meeting: 

- [x] Storage ordered; needs to be installed. @sbrunswi 
- [ ] Lab management: Tables etc. need to be labelled @ManthanKeim 
- [ ] Website update: New people to be added
- [ ] Access, usage of signature etc. 
- [ ] Create a guide for changing permission on tower - update on creating a tutorial/ wiki for everyone to use. @SahithiKasim 
- [ ] Schedule for upcoming presentations @crashidian 

## Software Dev/Github and other tools: 
- Tutorial Rclone

## IronHacks
- Update from Manthan and Harsha on development involvement
- Regular meeting times
- Job post for volunteer community manager; posting on twitter etc. 

## PhD and MS Thesis
- Manthan to present his master thesis proposal
- Sahithi and Harsha: Update plan of study finalization and committee members
- Exchange on classes related to AI/ML - which one were taken by the students etc. 

## Presentation on research projects/program and research learning 
- Operations efficiency UAV @NI-mi7 
- Empathy in conversational AI @yifan and @crashidian  and @sbrunswi 
- Sahithi: Hidden dependencies in OSS supply-chains and risk
- Discussion: Where can we help each other to learn new topics/methods? 

## Grants
- POSE 2023/09/07
- CISE 2023/09/30
- Smart and Connected Health 2023/11/09 with UCI
Still open for discussion: Where to submit the idea on LLMs and empathy (prompt engineering)

**Comment by crashidian** [comment]
**Action Items 8/28/2023**

- [ ] Lab management: Need man power to assist with the moving of furniture and moving of the computers.
- [ ] Lab management: renew academic license for organizations
- [ ] Lab management: update github team members. Add team's github's username to master contact sheet. 
- [ ] Website update: New people to be added
- [ ] Website update: All members need to get an ORCID for website. 
- [ ] Website update: Shatayu needs to be moved from Alumni to Active. Add Nevena to Active. See issue no. 76. Due by August 29, 2023. 
- [ ] Box access: check to see who needs access. 
- [ ] Guides: create a guide for rclone from Lionel's presentation. 
- [ ] AI Courses. Christopher added course descriptions to CSU Global in Chat. Syllabi attached here. Julie L. Carr, MA. Enrollment Counselor. Colorado State University Global. O: 303-653-9915. E: julie.carr@csuglobal.edu
- [ ] Plan of Study and Committee building for Sahithi, Christopher.
[CSC510_Foundations of Artificial Intelligence_Sample Syllabus.pdf](https://github.com/opendigital/opendigital-org/files/12457159/CSC510_Foundations.of.Artificial.Intelligence_Sample.Syllabus.pdf)
[CSC506_SampleStudentCourseSyllabus.pdf](https://github.com/opendigital/opendigital-org/files/12457161/CSC506_SampleStudentCourseSyllabus.pdf)
[CSC525_CourseSyllabus_Prinicples of Machin Learning_Sample Syllabus.pdf](https://github.com/opendigital/opendigital-org/files/12457162/CSC525_CourseSyllabus_Prinicples.of.Machin.Learning_Sample.Syllabus.pdf)

**Comment by RCODI** [comment]
Comments for Manthan's master thesis: 

1) Problem statement: 
Practical motivation (Slide 1): 
1.1. You motivate it via software programming more specifically; but you then talked about data science programmers; make sure that you clarify that you focus specifically focus on data science programming and how it is different from software engineering and batch-programming
1.2. Add also a scientific motivation: you do not really tell us what the literature has said before
1.3. Problem statement is practically motivated but than on slide 3/4 you talk about cognition
1.4. Purpose of study: It is not clear how you connect the problem statement to the purpose; also provide references about cognition and actually relate to performance versus understanding as cognition is a very broad literature stream!
1.5. Slide 7 - this is actually your literature review and the gaps in the literature and that should be presented before your "purpose of the study and what you are studying - so we should re-organize the paper; that is a good slide though. 
2. Literature review? Where is that in the heading?
3. What is your research method? You have to parts: The design of the recommendor and the experiment to validate it so that is design science! 
4. Proposed solution is on slide 9 - your jumping back and forth between design of the study and study of the recommendor and theory - so I think that is confusing the listener - so we need to restructure your slides accordingly; you have great ideas we just have to better organizing it. 
5. Are you sure the research questions are correct? 
6. I think the slides on the evaluation of recommendors is not that helpful but is rather confusing the listener! I think it is not clear what you are doing and how you code quality relates to cognition and performance versus understanding! 
7. Slides on types of bugs in Juptyer notebooks requires a citation

**Comment by NevenaIvanovic** [comment]
comments on the thesis: 
- thoughts on the experience of the sample considering the selection of bugs
- reduce the amount of text on the slides in general, more concise language
- problem statement is too long and has unnecessary words
- add screenshot examples of inefficient debugging behaviors (e.g. search queries)
- the video example is too fast, adjust your explanation/speech to the video/mouse movements in the video
- provide a chart/diagram showing different experimental groups/conditions
- research questions should appear much earlier in the presentation
- literature review us unclear, large diagram is too detailed so audience has no use of it since there is no time to read through it all. numbers in the diagram unclear
- use the usual research steps and label them clearly (motivation, research questions, literature, method etc.)
- use the same font, style throughout the presentation

**Comment by crashidian** [comment]
Chris Rashidian Comments on Presentation:

- The presentation had too many words and you were trying to jam too much information within each slide. The slide is to assist to convey the point, not guide the point. You are the expert of your subject, not the slides.
- I would like to see more visualizations on your slides (i.e., screenshots, charts, diagrams)
- Your RQ was too late in the presentation
- You were speaking way too fast and it was hard to follow. Speak slower so others can easily follow.
